meta-webos/recipes-graphics/xorg-lib/libxkbcommon_%.bbappend

```diff
@@ -0,0 +1,2 @@
+BBCLASSEXTEND = "native nativesdk"
```

Comment:
I guess its also unnecessary if we do not change the dependency of libsdl2-native.